benchmark/README.md

```diff
@@ -0,0 +1,47 @@
+# Benchmarks
```

Comment:
May ask you to add a section to **Benchmark** in the **README** file? I think that it could be placed after **Debug** section.